tutorial/01_image_filtering.md

```diff
@@ -532,31 +576,9 @@ def bounded_slice(center, xy_max, size=1, i_min=0):
 ```
 
 ```{code-cell} ipython3
-mean_filter_interactive_demo(bright_square)
```

Comment:
I can't figure out why the first image in this interactive example is using grayscale. Also weird that the square in the middle isn't white but light gray. Probably not a huge issue but I'd like to be able to explain code I might be talking about. :sweat_smile: 

![image](https://github.com/scipy-2023-image-analysis/tutorial/assets/20140352/0c48fc82-b3e2-478c-9e72-56adfac98152)

Reply:
I don't know either! This demo was created by Tony Yu for the 2014 (!) tutorial and remains pretty much unchanged since then. 😂 Every year I think "I should update it" and then every year it's the Friday before the tutorial again! 🤣